packages/web-react/src/components/UNSTABLE_Toggle/__tests__/UNSTABLE_Toggle.test.tsx

```diff
@@ -0,0 +1,92 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { classNamePrefixProviderTest } from '../../../../tests/providerTests/classNamePrefixProviderTest';
+import { validationStatePropsTest } from '../../../../tests/providerTests/dictionaryPropsTest';
+import { requiredPropsTest } from '../../../../tests/providerTests/requiredPropsTest';
+import { restPropsTest } from '../../../../tests/providerTests/restPropsTest';
+import { stylePropsTest } from '../../../../tests/providerTests/stylePropsTest';
+import UNSTABLE_Toggle from '../UNSTABLE_Toggle';
+
+describe('UNSTABLE_Toggle', () => {
+  classNamePrefixProviderTest(UNSTABLE_Toggle, 'UNSTABLE_Toggle');
+
+  stylePropsTest(UNSTABLE_Toggle);
+
+  restPropsTest(UNSTABLE_Toggle, 'input');
+
+  validationStatePropsTest(UNSTABLE_Toggle, 'UNSTABLE_Toggle--');
+
+  requiredPropsTest(UNSTABLE_Toggle, 'checkbox', 'id', 'example-id');
+
+  it('should have correct className', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" />);
+
+    expect(screen.getByRole('checkbox').parentElement).toHaveClass('UNSTABLE_Toggle');
+  });
+
+  it('should have label classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" />);
+
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label');
+    expect(screen.getByText('Toggle Label')).toContainHTML('label');
+  });
+
+  it('should have label with required classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isRequired />);
+
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label');
+    expect(screen.getByText('Toggle Label')).toHaveClass('UNSTABLE_Toggle__label--required');
+  });
+
+  it('should have hidden classname', () => {
+    render(<UNSTABLE_Toggle id="test-toggle" label="Toggle Label" isLabelHidden />);
+
+    const label = screen.getByText('Toggle Label');
+
+    expect(label).toHaveClass('UNSTABLE_Toggle__label');
+    expect(label).toHaveClass('UNSTABLE_Toggle__label--hidden');
```

Comment:
why here you have two expect for two classes and below you have only one, but for two classes? :)

Reply:
Mistake :D